op-batcher/batcher/service.go

```diff
@@ -139,14 +137,14 @@ func (bs *BatcherService) initFromCLIConfig(ctx context.Context, version string,
 	opts = append(optsFromRPC, opts...)
 
 	// MultipleNodesClient requires at least 2 URLs.
```

Comment:
Should we do a length check on `QueryServiceURLs`, similar to what we did on `EspressoUrls`?

Reply:
MultipleNodesClient does the check on initialisation, so this would be a duplicate check. I'll remove the comment